### PR TITLE
PHPStan: some fixes

### PR DESCRIPTION
**Pull request by sreichel**
k/t

### CODE REVIEW

File: app/code/core/Mage/Paypal/Model/Payflowlink.php [hunk 1]
```diff
@@ -365,7 +365,6 @@ protected function _getOrderFromResponse()
     /**
      * Build request for getting token
      *
-     * @param Mage_Sales_Model_Order_Payment|Mage_Payment_Model_Info $payment
      * @return Varien_Object
```
Comment (kiatng) [inline app/code/core/Mage/Paypal/Model/Payflowlink.php:368]:
```suggestion
     * @return Mage_Paypal_Model_Payflow_Request
```

**Comment by fballiano** [comment]
what does "k/t" mean?

**Review by addison74 — APPROVED** [review]

File: app/code/core/Mage/SalesRule/Model/Quote/Discount.php [hunk 2]
```diff
@@ -134,7 +134,7 @@ protected function _aggregateItemDiscount($item)
     /**
      * Add discount total information to address
      *
-     * @return $this
+     * @return $this|array
```
Comment (kiatng) [inline app/code/core/Mage/SalesRule/Model/Quote/Discount.php:137]: I cannot see how it can return an array.

Comment (sreichel) [reply app/code/core/Mage/SalesRule/Model/Quote/Discount.php:137]: To match `parrent::fetch(()`, that returns an array.

**Review by kiatng — APPROVED** [review]

**Pull request merged**